feel/feeldiscr/meshstructured.cpp

```diff
@@ -72,6 +78,40 @@ MeshStructured::MeshStructured( int nx, int ny, double pixelsize, holo3_image<fl
     std::map<int,int> idStructuredMeshToFeelMesh;
     node_type coords( 2 );
 
+    
+    // polygon build
+    std::vector<polygon_type> list_poly ;
+    //std::cout << list_poly.size() << std::endl;
+    polygon_type poly;
+    bool inPoly = false;
+    bool testInPoly = false;
+    if (withPoly)
```

Comment:
@LANTZT is this (polygonal extraction) described in the documentation of the holo3 project ?